lib/composables/Error.ts

```diff
@@ -0,0 +1,185 @@
+/**
+ * Adapted from
+ * @see https://github.com/vuejs/core/blob/76c9c742e9b045d1342f5952866972498e59f00b/packages/runtime-core/src/component.ts
+ * @see https://github.com/vuejs/core/blob/bc37258caa2f6f67f4554ab8587aca3798d92124/packages/runtime-core/src/warning.ts
+ * @see https://github.com/getsentry/sentry-javascript/blob/2cfb0ef3fa5c40f90c317267a4d10b969994d021/packages/vue/src/errorhandler.ts
+ *
+ * Original licenses:
+ *
+ * (c) 2018-present Yuxi (Evan) You and Vue contributors
+ * @license MIT
+ *
+ * (c) 2019 Sentry (https://sentry.io) and individual contributors
+ * @license MIT
+ */
+
+import * as Sentry from '@sentry/browser'
+import { type User } from '@sentry/types'
+import { consoleSandbox } from '@sentry/utils'
+import { pauseTracking, resetTracking } from '@vue/reactivity'
+import {
+  type ComponentInternalInstance,
+  type ComponentOptions,
+  type ComponentPublicInstance,
+  type ConcreteComponent,
+  type MaybeRefOrGetter,
+  type VNode,
+  toValue
+} from 'vue'
+import { useError } from '../stores/Error'
+import { isFunction } from '../support/Utils'
+
+export { useError } from '../stores/Error'
```

Comment:
Should we import stores as store, or wrap with composable. I would like to discuss it a bit. I'm always not sure... but these days thinking, maybe it would be just OK to import it from where it belongs 🤔  So import from `@/stores/Error` instead of `@/composables/Error`.

What do you think? Then we need to import `useErrorHandler` and `useError` from different path, but... I don't know. But the upside is that it makes it clear that `useError` is "global" state.

Reply:
Yeah makes sense I guess. We won't be importing both useErrorHandler and useError in same file anyway.

Reply:
Nice. OK let's do that. So, remove this re-export and I think we are good to go 🔥